resources/assets/config.json

```diff
@@ -14,8 +14,8 @@
   "cacheBusting": "[name]_[hash:8]",
   "watch": [
     "app/**/*.php",
+    "app/controllers/**/*.php",
```

Comment:
No need for this, the line above has everything covered

Reply:
did not dare to delete)